widgetastic_manageiq.py

```diff
@@ -878,3 +878,144 @@ def last_page(self):
     @property
     def items_amount(self):
         return self.paginator.page_info()[1]
+
+
+class Stepper(Widget):
+    """ A CFME Stepper Control
+
+        stepper = Stepper(locator='//div[contains(@class, "timeline-stepper")]')
+        stepper.inrease()
+    """
+    def __init__(self, parent, locator, logger=None):
+        super(Stepper, self).__init__(parent=parent, logger=logger)
+
+        self._stepper_control = self.browser.element(locator, parent=parent)
+
+        self.minus_button = Button(self, '-')
+        self.plus_button = Button(self, '+')
+        self.value_field = Input(self, locator='.//input[contains(@class, '
+                                               '"bootstrap-touchspin")]')
+
+    def __locator__(self):
+        return self._stepper_control
+
+    def read(self):
+        return int(self.value_field.read())
+
+    def decrease(self):
+        self.minus_button.click()
+
+    def increase(self):
+        self.plus_button.click()
+
+    def set_value(self, value):
+        if value < 1:
+            raise ValueError('The value cannot be less than 1')
+        steps = int(value) - self.read()
+        if steps == 0:
+            return
+        elif steps > 0:
+            operation = self.increase
+        else:
+            operation = self.decrease
+
+        steps = abs(steps)
+        for step in range(steps):
+            operation()
+
+    def fill(self, value):
+        self.set_value(value)
+
+
+class RadioButton(Button):
+    """CFME Radio Button Control
+       it should be mainly used in RadioGroup control
+    """
+    def __init__(self, parent, name, logger=None):
+        super(RadioButton, self).__init__(parent=parent, logger=logger)
+        self._name = name
+
+    def __locator__(self):
+        return (
+            './/*[self::input and @type="radio" '
+            'and parent::label[normalize-space(.)="{n}"]]'.format(n=self._name))
+
+    @property
+    def selected(self):
+        if 'ng-valid-parse' in self.browser.classes(self):
+            return True
+        return False
+
+    @property
+    def name(self):
+        return self._name
+
+
+class RadioGroup(Widget):
+    """ CFME Radio Group Control
+
+        rg = RadioGroup(locator='//span[contains(@class, "timeline-option")]')
+        rg.
+    """
+    def __init__(self, parent, locator, logger=None):
+        super(RadioGroup, self).__init__(parent=parent, logger=logger)
+        self._locator = locator
+
+    def __locator__(self):
+        return self._locator
+
+    @property
+    def button_names(self):
+        buttons = self.browser.elements('//label[input[@type="radio"]]', parent=self)
```

Comment:
And I do believe that with recent widgetastic update unnecessary as self.browser should figure the parent out automatically. Can you try?